data-error/Cargo.toml

```diff
@@ -10,7 +10,7 @@ bench = false
 
 [dependencies]
 thiserror = "1"
-reqwest = "0.11.11"
+reqwest = { version = "0.11.11", features = [ "json", "rustls-tls" ], default-features = false }
```

Comment:
@kirillt, this makes `OpenSSL` not required when compiling/building the project.

It's related to [Can't cross-compile project with OpenSSL](https://users.rust-lang.org/t/cant-cross-compile-project-with-openssl/70922/3)